test/ResourceManager.test.cpp

```diff
@@ -3,22 +3,20 @@
 #include <stdexcept>
 
 
-TEST(ResourceManager, RefusesAbsolutePaths) {
+TEST(ResourceManager, ConstructResourceManager) {
 	ResourceManager resourceManager("");
 
+	// Refuse absolute paths
 #ifdef _WIN32
 	EXPECT_THROW(resourceManager.GetResourceStream("C:/Archive.vol"), std::runtime_error);
 	EXPECT_THROW(resourceManager.GetResourceStream("C:/PathTo/Archive.vol"), std::runtime_error);
 #endif
 	EXPECT_THROW(resourceManager.GetResourceStream("/Archive.vol"), std::runtime_error);
 	EXPECT_THROW(resourceManager.GetResourceStream("/PathTo/Archive.vol"), std::runtime_error);
```

Comment:
These tests don't seem related to `ConstructResourceManager`. It's testing the `GetResourceStream` method on an already constructed `ResourceManager` object. They should probably be kept separate from the other tests, with the old test name restored for this portion (and possibly also mentioning the method name somewhere).

Reply:
Oh yeah. I misread what these tests were for. You are right they should be separated.